README.md

```diff
@@ -37,7 +37,7 @@ npm install --save-dev eslint eslint-plugin-regexp
 > **Requirements**
 >
 > - ESLint v6.0.0 and above
-> - Node.js v8.10.0 and above
+> - Node.js v12.x, v14.x and above
```

Comment:
It might be nice to write that down in [`package.json`](https://docs.npmjs.com/cli/v7/configuring-npm/package-json#engines) as well.

Reply:
Good idea! I added it.